azure-pipelines.yml

```diff
@@ -196,13 +196,14 @@ jobs:
     pool:
       vmImage: macos-11
     steps:
-      - task: UseDotNet@2
-        displayName: 'Install .NET SDK'
+      - task: CmdLine@2
+        displayName: Install Nightly .NET Build
         inputs:
-          version: $(NETCORE_VERSION)
-          includePreviewVersions: false
+          script: |
+            curl -O https://aka.ms/dotnet/6.0/daily/dotnet-sdk-osx-x64.pkg
```

Comment:
looks like the path should be https://aka.ms/dotnet/6.0.2xx/daily/dotnet-sdk-osx-x64.pkg